src/bodyElement.ts

```diff
@@ -143,6 +152,8 @@ function toSerialisable(elem: BodyElement): JsonSerialisable {
             return { ...elem, content: serialiseNodes(elem.content) };
         case ElementKind.MediaAtom:
             return { ...elem, caption: map(serialiseFragment)(elem.caption) };
+        case ElementKind.Callout:
+            return { ...elem, campaign: serialiseCampaign(elem.campaign) };
```

Comment:
I don't think it is, but I don't think Callouts are used in lilveblogs so it worth adding? Would it just be stringifying the Campaign object?

Reply:
Everything [here](https://github.com/guardian/apps-rendering/blob/30290357ca9b93f6bf49e342a16c19c474dca2ba/api-models/thrift/src/main/thrift/appsRendering.thrift#L40) is a `string`, `bool`, `number`, `Array` or `Object` isn't it? If so, won't JSON.stringify automatically be able to convert it?

Essentially what I'm asking is: if you added `case ElementKind.Callout:` to the line after `case ElementKind.Embed:`, would it "just work"?

Reply:
That doesn't work, there's a few nested types that would need serialising https://github.com/guardian/apps-rendering/blob/30290357ca9b93f6bf49e342a16c19c474dca2ba/api-models/thrift/src/main/thrift/appsRendering.thrift#L47

Reply:
Ok, think this is because `Int64` gets represented as a `class` once it's been converted to TypeScript. One solution to this would be to add this case to the `JsonSerialisable` type:

```ts
| { toString(): string }
```

Reply:
Following a discussion offline, this is probably fine as-is, although if Callouts start to appear in liveblogs we may have to re-visit.